bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -264,7 +264,10 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
     if (minBatchSize > maxReasonableMinBatchSize) minBatchSize = maxReasonableMinBatchSize;
```

Comment:
Do we have an issue number for this effort? If positive, please mention it in the description, otherwise please create it first.

Reply:
done